client/me/purchases/payment-methods/components/payment-method-edit-button.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button } from '@automattic/components';
+import { FunctionComponent } from 'react';
```

Comment:
Minor: this could be `import type`.
```suggestion
import type { FunctionComponent } from 'react';
```

Reply:
Fixed.